src/osd/OSD.cc

```diff
@@ -7947,6 +7947,7 @@ void OSD::osdmap_subscribe(version_t epoch, bool force_request)
 
 void OSD::trim_maps(epoch_t oldest, bool skip_maps)
 {
+  // possible leakage: https://tracker.ceph.com/issues/61962
```

Comment:
Always summarize the issue inline so that readers don't need to consult the tracker to understand what's going on.  Do also include the tracker link though